testing/bear/APKBUILD

```diff
@@ -0,0 +1,38 @@
+# Contributor: Orson Teodoro <orsonteodoro@hotmail.com>
+# Maintainer: Orson Teodoro <orsonteodoro@hotmail.com>
+pkgname=bear
+_pkgname=Bear
+pkgver=2.3.11
+pkgrel=0
+pkgdesc="Build EAR - Bear is a tool that generates a compilation database for clang tooling."
+url="https://github.com/rizsotto/Bear"
+arch="noarch"
+license="GPL-3.0"
+makedepends="cmake py3-lit"
+options="check !archcheck"
```

Comment:
This legitimately contains arch-specific binaries (`libear.so`), so it should be `arch="all"` and `!checkroot` should be removed.